test/browser/render.js

```diff
@@ -461,13 +462,13 @@ describe('Rendering', function () {
           'my-element': Fixture89.myElement
         })
       });
-
       responses.DOM.select(':root').observable.skip(1).take(1).subscribe(function (root) {
         setTimeout(() => {
           let myelement = root.querySelector('.myelementclass');
           assert.notStrictEqual(myelement, null);
           assert.strictEqual(myelement.tagName, 'H3');
           assert.strictEqual(myelement.textContent, '123');
+          done()
```

Comment:
Why this `done()`? Notice that `done()` must be called once and it should be called in the second timeout below.